mathlibtools/lib.py

```diff
@@ -191,10 +191,36 @@ def lean_version_toml(version: VersionTuple) -> str:
         return ver_str
     else:
         return 'leanprover-community/lean:' + ver_str
-    
+
+
+def clean(dir):
+    print('cleaning {} ...'.format(dir))
+    for (path, _dirs, fs) in os.walk(dir):
+        for fn in fs:
+            if os.path.splitext(fn)[1] == '.olean':
+                fn = Path(path, fn)
+                os.remove(fn)
+
+
+def delete_zombies(dir):
```

Comment:
This seems very complicated for no reason. Did you have a look at how we check oleans are older than leans in the check function?